src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -577,7 +577,7 @@ case class WorkflowActor(workflow: WorkflowDescriptor,
         self ! CallFailed(collector, None, e.getMessage)
       case Success(outputs) =>
         log.info(s"Collection complete for Scattered Call ${collector.scope.fullyQualifiedName}.")
-        self ! CallCompleted(collector, outputs)
+        self ! CallCompleted(collector, outputs, 0)
```

Comment:
Oh sure, you make me put constants in for all literals and look at this!

(Don't actually change it)